addons/account/tests/test_chart_template.py

```diff
@@ -109,13 +108,6 @@ def _tax_vals(name, amount, tax_tag_id=None, children_tax_xmlids=None, active=Tr
     }
     if children_tax_xmlids:
         tax_vals.update({'children_tax_ids': [Command.set(children_tax_xmlids)]})
-    else:
-        tax_vals.update({'repartition_line_ids': [
-            Command.create({'document_type': 'invoice', 'factor_percent': 100, 'repartition_type': 'base', 'tag_ids': tag_command}),
-            Command.create({'document_type': 'invoice', 'factor_percent': 100, 'repartition_type': 'tax'}),
-            Command.create({'document_type': 'refund', 'factor_percent': 100, 'repartition_type': 'base'}),
-            Command.create({'document_type': 'refund', 'factor_percent': 100, 'repartition_type': 'tax'}),
-        ]})
```

Comment:
Why this change ?

Reply:
The tax restriction after usage creates issues in the chart template
tests: some of those test would re-create the repartition
lines that were created by default after the tax is used which is not
permitted anymore. This removes the Command create that would recreate the lines. When tags are needed (only in 1 test), the test sets it itself.